app/models/defendant.rb

```diff
@@ -98,11 +98,7 @@ def valid_maat_reference?
   end
 
   def _maat_reference
-    refs = offences.map(&:maat_reference).uniq.compact
-
-    raise(Errors::DefendantError, "Too many maat references: #{refs}") if refs.size > 1
-
-    refs&.first
+    LaaReference.find_by(defendant_id: id, linked: true)&.maat_reference
```

Comment:
Note that this doesn't help with the v2 case summary endpoint, which doesn't pass back this field and instead just passes back the offence summaries, and CDAPI pulls out the first of these and uses it as "the" MAAT number. So we would need to change CDAPI to use this canonical MAAT reference when calling that endpoint otherwise the MAAT ID could appear differently on the case summary page vs the defendant page.